### PR TITLE
wip redirect after login show user info

### PR DESCRIPTION
**Pull request by TeriPastorino**
@levicole

### CODE REVIEW

File: db/hello:1:greeter_spec.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+hello/1/greeter_spec.rb
```
Comment (levicole) [inline db/hello:1:greeter_spec.rb]: I'm not sure what's going on here, but I have a feeling making this file was an accident?

Comment (TeriPastorino) [inline db/hello:1:greeter_spec.rb]: indeed i think was working on something else! will delete, i sort of saw it yesterday and thought hmmm 

----- Original Message -----

File: spec/features/user_registration_spec.rb [hunk 2]
```diff
@@ -69,40 +70,40 @@ def signup(username, email, password)
...
-    visit user_session_path #visit users/sign_in page
-    fill_in 'Email',    with: user.email
-    fill_in 'Password', with: user.password
-    click_button "Sign In"
-    expect(current_path).to eq "/"
-    expect(page).to have_content("Hello")
+  describe "successfully" do
+    let!(:user) { FactoryGirl.create(:user) } 
+    it "displays a flash message indicating success" do
+      visit user_session_path #visit users/sign_in page
```
Comment (levicole) [inline spec/features/user_registration_spec.rb:76]: So, I was supposed to get you this:

https://github.com/plataformatec/devise/wiki/How-To:-Test-with-Capybara

This is how you log in without having to go through all the motions.

you will probably want to include the Warden::Test::Helpers module in the rails_helper file that's in your spec directory. But only for feature specs. Let me know if you need help figuring out how to do that. I can point you in the right direction.

Comment (levicole) [inline spec/features/user_registration_spec.rb:76]: Also, we can just keep track of all the things we want to refactor and then have a refactoring session next week if you want! :)

**Comment by levicole** [comment]
I think we are good to go on this one! :+1:

File: spec/features/user_registration_spec.rb (hunk 2 above)
Comment (TeriPastorino) [inline spec/features/user_registration_spec.rb:76]: Ok I will look at this one and that might be a good idea based in where I am in project in general

I will make appropriate changes and merge

**Pull request merged**